### PR TITLE
Actor details screen

### PR DESCRIPTION
**Pull request by amarburic**
PR includes changes to movie and tv show detail screens as well, since they share common base class and some layouts. Feedback on swipe refresh layout needed.

### CODE REVIEW

File: app/src/main/java/ba/abh/internship/amarburic/popularmovies/ui/fragments/DetailsFragment.java [hunk 1]
```diff
@@ -20,9 +21,11 @@
 
 public abstract class DetailsFragment extends Fragment implements UpdateBaseObjectCallback {
 
+    Long baseObjectId;
     BaseObject baseObject;
     ScrollView scrollView;
     Pair<Integer, Integer> scrolledScrollView;
+    SwipeRefreshLayout swipeRefreshLayout;
     boolean uiSetup;
```
Comment (farukatlant) [inline app/src/main/java/ba/abh/internship/amarburic/popularmovies/ui/fragments/DetailsFragment.java]: Shouldn't these variables be private?

File: app/src/main/java/ba/abh/internship/amarburic/popularmovies/ui/fragments/DetailsFragment.java [hunk 2]
```diff
@@ -20,9 +21,11 @@
 
 public abstract class DetailsFragment extends Fragment implements UpdateBaseObjectCallback {
 
+    Long baseObjectId;
```
Comment (farukatlant) [inline app/src/main/java/ba/abh/internship/amarburic/popularmovies/ui/fragments/DetailsFragment.java]: Base object already has an id

File: app/src/main/res/layout/card_view_filmography.xml [hunk 3]
```diff
@@ -23,22 +23,22 @@
             <TextView
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:ellipsize="none"
-                android:scrollHorizontally="false"
+                android:ellipsize="end"
+                android:layout_gravity="center_horizontal"
+                android:singleLine="true"
```
Comment (farukatlant) [inline app/src/main/res/layout/card_view_filmography.xml]: singleLine is deprecated, we should use lines="1"

File: app/src/main/res/layout/card_view_filmography.xml [hunk 4]
```diff
@@ -23,22 +23,22 @@
...
-                android:layout_gravity="center_horizontal"
                 android:id="@+id/text_view_film_name"/>
             <TextView
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:ellipsize="none"
-                android:scrollHorizontally="false"
+                android:ellipsize="end"
+                android:layout_gravity="center_horizontal"
+                android:singleLine="true"
```
Comment (farukatlant) [inline app/src/main/res/layout/card_view_filmography.xml]: singleLine is deprecated, we should use lines="1"

File: app/src/main/res/layout/card_view_filmography.xml [hunk 5]
```diff
@@ -23,22 +23,22 @@
...
                 android:background="@color/colorCardBackground"
                 style="@style/DetailsBrightText"
                 android:text=""
```
Comment (farukatlant) [inline app/src/main/res/layout/card_view_filmography.xml]: not neaded

File: app/src/main/res/layout/details_main_info_actor.xml [hunk 6]
```diff
@@ -47,12 +47,14 @@
             style="@style/DetailsBrightText"
             android:text="@string/text_date_of_birth"
             android:paddingRight="10dp"
+            android:id="@+id/text_view_details_actor_date_of_birth_label"
             android:layout_row="0"
```
Comment (farukatlant) [inline app/src/main/res/layout/details_main_info_actor.xml]: we are using wrong namespace here: http://prntscr.com/h5qv4m

File: app/src/main/res/layout/filmography_list.xml [hunk 7]
```diff
@@ -1,21 +1,27 @@
...
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/text_filmography"
+            android:paddingBottom="10dp"
+            style="@style/DetailsBrightText"/>
+        <android.support.v7.widget.RecyclerView android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:background="@color/colorCardBackground"
+            android:id="@+id/recycler_view_filmography"
+            xmlns:android="http://schemas.android.com/apk/res/android" />
```
Comment (farukatlant) [inline app/src/main/res/layout/filmography_list.xml]: schemas should be in root layout of layout file

File: app/src/main/java/ba/abh/internship/amarburic/popularmovies/ui/fragments/DetailsFragment.java [hunk 8]
```diff
@@ -61,12 +71,25 @@ public void onDestroyView() {
 
     @Override
     public void updateBaseObject(BaseObject baseObject) {
+        while(!uiSetup);
```
Comment (farukatlant) [inline app/src/main/java/ba/abh/internship/amarburic/popularmovies/ui/fragments/DetailsFragment.java]: This will block main thread

Comment (amarburic) [reply app/src/main/java/ba/abh/internship/amarburic/popularmovies/ui/fragments/DetailsFragment.java]: updateBaseObject is now being called from a different thread.

**Pull request merged**